src/net/sourceforge/kolmafia/RequestEditorKit.java

```diff
@@ -1898,31 +1902,29 @@ private static void addChoiceSpoilers(final String location, final StringBuffer
         StringBuilder spoilerBuffer = new StringBuilder(spoiler.toString());
 
         // If this decision has an item associated with it, annotate it
-        if (spoiler instanceof ChoiceManager.Option) {
-          ChoiceManager.Option option = ((ChoiceManager.Option) spoiler);
-          AdventureResult[] items = option.getItems();
-
-          // If this decision leads to one or more item...
-          for (int it = 0; it < items.length; it++) {
-            AdventureResult item = items[it];
-            if (item != null) {
-              if (it > 0) {
-                spoilerBuffer.append(" or ");
-                spoilerBuffer.append(item.getName());
-              }
-
-              // List # in inventory
-              spoilerBuffer.append(
-                  "<img src=\"/images/itemimages/magnify.gif\" valign=middle onclick=\"descitem('");
-              spoilerBuffer.append(ItemDatabase.getDescriptionId(item.getItemId()));
-              spoilerBuffer.append("');\">");
-
-              int available = KoLCharacter.hasEquipped(item) ? 1 : 0;
-              available += item.getCount(KoLConstants.inventory);
-
-              spoilerBuffer.append(available);
-              spoilerBuffer.append(" in inventory");
+        ChoiceAdventures.Option option = ((ChoiceAdventures.Option) spoiler);
```

Comment:
probably don't need the cast here? spoiler is already the right type?

Reply:
I actually don't need the temporary variable at all; it is only used in the next line to call getItems() on.